gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LazyCreateStartScriptTask.java

```diff
@@ -37,12 +37,14 @@ public LazyCreateStartScriptTask() {
         return mainClassName;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Is something in here going to be deprecated in Gradle 8?

Reply:
The `setMainClassName` on the parent class is deprecated in favor of setting the value through the property.
I think it's the easiest to also remove this method once the parent method gets removed in Gradle 8 (?).